app/views/snacks/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%=@snack.name%></h1>
+<h2><%=@snack.price%></h2>
+<h3>Locations:</h3>
+<% @snack.machines.each do |machine| %>
+  <p> <%= machine.location %>Average Price: <%=machine.average_price %></p>
```

Comment:
you can look into using `number_to_currency` as an action view helper to help you format prices